packages/dredd/test/integration/regressions/regression-531-test.js

```diff
@@ -0,0 +1,62 @@
+import { assert } from 'chai';
+
+import { runDreddWithServer, createServer } from '../helpers';
+import Dredd from '../../../lib/Dredd';
+
+describe(`Sending and receiving "application/hal+json" according to produces/consumes in OpenAPI 2.0`, () => {
+  let runtimeInfo;
+
+  before((done) => {
+    const app = createServer();
+    app.get('/articles', (req, res) =>
+      res
+        .set({
+          'content-type': 'application/hal+json',
+        })
+        .json([
+          {
+            id: 1,
+            title: 'Creamy',
+            text: 'Arbitrary',
+            _links: {
+              self: {
+                href: '/articles/1',
+              },
+            },
+          },
+        ]),
+    );
+    app.post('/articles', (req, res) => {
+      res.status(201).json({
+        id: 2,
+        title: 'Crispy schnitzel',
+        text: 'Prepare eggs...',
+      });
+    });
+
+    const dredd = new Dredd({
+      options: {
+        path: './test/fixtures/531-produces-consumes.yaml',
```

Comment:
Should we use `path.join`? wondering if this will break on windows and other platforms where the path separator is not `/` but something like `\`.

Reply:
There are other tests that provide `path` option like this. I'll check the source code if there's any `path.normalize(options.path)` or similar to be performed on the Dredd's side.

If not, let's raise this as a new issue and adjust all values of `path`. What do you think?

Reply:
I've went through Dredd's codebase and haven't found a place where it would have normalized the `options.path`. Raising an issue: https://github.com/apiaryio/dredd/issues/1653